futures-util/src/compat/tokio.rs

```diff
@@ -0,0 +1,71 @@
+use crate::{future::FutureExt, try_future::TryFutureExt};
+use std::{
+    future::FutureObj,
+    task::{Executor, SpawnErrorKind, SpawnObjError},
+};
+use tokio_executor::{DefaultExecutor, Executor as _};
+
+/// An executor that delegates to `tokio`'s [`DefaultExecutor`], will panic if
+/// used in the context of a task that is not running on `tokio`'s executor.
+///
+/// # Examples
+///
+/// ```ignore
+/// #![feature(async_await, await_macro, underscore_imports, pin)]
```

Comment:
Is the `underscore_imports` feature required?

Reply:
Nope, I just try and use minor unstable features to give them a good testing. I’ll remove it.